docs/source/search/_intro.rst

```diff
@@ -0,0 +1,14 @@
+Indico’s version 3.0 introduced a brand new reusable and agnostic search module
```

Comment:
I think we use `'` ASCII quotes as apostrophes everywhere else

Reply:
I don't think standalone "agnostic"  works; something is usually "something-agnostic", e.g. "backend-agnostic" or "database-agnostic"...